modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/service/test/UserServiceTest.java

```diff
@@ -96,6 +97,57 @@ public void testAddInvalidOrganizationUsers() throws Exception {
 		}
 	}
 
+	@Test
+	public void testAddUserToOrgWithoutPermission() throws Exception {
```

Comment:
probably is better `testUpdateOrganizationsWithoutPermission()` because it's the name of the method you're testing